debezium-connector-mysql/src/test/java/io/debezium/connector/mysql/AbstractMysqlDefaultValueTest.java

```diff
@@ -339,7 +339,10 @@ public void parseTimeDefaultValue() {
                 "  I datetime(3) NOT NULL DEFAULT '0000-00-00 00:00:00.000'," +
                 "  J datetime NOT NULL DEFAULT '2018-06-26 12:34:56'," +
                 "  K datetime(3) NOT NULL DEFAULT '2018-06-26 12:34:56.000'," +
-                "  L datetime(2) NOT NULL DEFAULT '2018-06-26 12:34:56.78'" +
+                "  L datetime(2) NOT NULL DEFAULT '2018-06-26 12:34:56.78'," +
+                "  M datetime NOT NULL DEFAULT '2000-01-00 00:00:00'," +
```

Comment:
So to make sure I get this right: all these values result in a default value of the epoch, as they are invalid essentially?

Reply:
Yes. The basic idea is the timestamp need not be "ALL" zeros. Sometimes, only month part can be zero OR only day part can be zero etc. All of them are invalid. So, If the column is optional, we use null. If the column is non optional, we use EPOCH timestamp.